adapters/repos/db/aggregator/date.go

```diff
@@ -191,6 +200,7 @@ func (a *dateAggregator) Median() string {
 
 // turns the value counter into a sorted list, as well as identifying the mode
 func (a *dateAggregator) buildPairsFromCounts() {
+	a.pairs = a.pairs[:0] // clear out old values in case this function called more than once
```

Comment:
is it safe to this? there will never be a situation the `len(a.pairs) == 0`?

Reply:
len(0) should not be a problem, but if you call it without adding a pair first it would probably panic.

I added creating an empty list in the constructor so a.pairs is never nil